Don't turn SSL into lowercase

TacticalCatto opened the pull request:
This pull request fixes pterodactyl/panel#3816.

mrxbox98 committed:
Fix Upper Case SSL Path

Commit:
Don't turn SSL into lowercase

mrxbox98 commented:
@TacticalCatto I created a PR on your branch which merges https://github.com/pterodactyl/wings/pull/113 into it that also fixes upper case ssl for letsencrypt files.

Commit:
Merge pull request #1 from mrxbox98/SSLFix

Fix Upper Case SSL Path in config

Commit:
Remove redundant code

This logic pretty much do nothing now, previously used for turning SSL into lowercase.

Commit:
Remove redundant dependencies

DaneEveritt commented on router/router_system.go:
```diff
@@ -103,15 +102,7 @@ func postUpdateConfiguration(c *gin.Context) {
-	// want to override that.
-	//
-	// If you pass through manual locations in the API call this logic will be skipped.
-	if strings.HasPrefix(cfg.Api.Ssl.KeyFile, "/etc/letsencrypt/live/") {
```
This code should not have been removed, the comment explains what it was doing, the only change that should have happened was removing the to lower call.

Without this every config update will overwrite any custom SSL configuration the user might have added.

Commit:
Revert
